components/automate-gateway/api/scripts/swagger.go

```diff
@@ -49,13 +49,13 @@ func main() {
 			if isPrefix || err != nil {
 				logrus.Fatalf("Failed with isPrefix=%t err=%v", isPrefix, err)
 			}
-			writer.WriteString(strings.Replace(string(line), "`", "` + \"`\" + `", -1))
-			writer.WriteString("\n")
+			writer.WriteString(strings.Replace(string(line), "`", "` + \"`\" + `", -1)) // nolint: errcheck
```

Comment:
😕 Is this script the same as the one above? What is going on here? ❓ (Not part of your PR, just confused)

Reply:
LOL no idea actually. Hmmmmmm.